universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockPredicatesEditView.java

```diff
@@ -109,9 +111,22 @@ private Modal predicateFormModal(
         .build();
   }
 
+  // TODO(#322): Create POST action endpoint for this form.
   private Tag renderPredicateForm(
       String blockName, QuestionDefinition questionDefinition, Tag csrfTag) {
 
+    ImmutableList.Builder<SimpleEntry<String, String>> actionOptionsBuilder =
+        ImmutableList.builder();
+    for (PredicateAction action : PredicateAction.values()) {
+      actionOptionsBuilder.add(new SimpleEntry<>(action.toDisplayString(), action.name()));
+    }
```

Comment:
When #1348 is submitted, this will end up something like

```java
Arrays.stream(PredicateAction.values()).collect(toImmutableMap(PredicateAction::toDisplayString, PredicateAction::name));
```

Reply:
Heads up that #1348 is merged!